### DISCUSSION TITLE
The relation column does not display values

### DISCUSSION THREAD

**Discussion by muksidin** [Budibase/budibase#11967] [Help] [answered]
Hi, after i upgrade the latest version, now the relationship column does not display the value.
[
![Screenshot_4](https://github.com/Budibase/budibase/assets/5665115/db4d4359-5bd6-4a97-839f-9079922c6cd7)
](url)

![Screenshot_5](https://github.com/Budibase/budibase/assets/5665115/51e3a308-b987-4e60-a4aa-dc04406996e1)

**Comment by ConorWebb96** [comment, marked as answer]
Hey @muksidin,

Am I correct in assuming that you are using an external datasource? This should work completely fine with the internal Budibase data source as it was working whenever I was testing.

I did however, replicate this on an external datasource e.g. Postgres database. I have wrote up an issue which can be found [here](https://github.com/Budibase/budibase/issues/11972).

I hope this helps.

**Reply by muksidin** [reply]
yes, i hope this is fixed soon. thanks
